apps/jenkins/jenkins/sbox-intsvc/jenkins-azure-vm-agent.yaml

```diff
@@ -30,73 +99,13 @@ spec:
-                          jenkinsssh soft nofile 40960
-                          jenkinsssh hard nofile 40960
-                          jenkinsssh soft
-                          proc 32768
-                          jenkinsssh hard nproc 32768
-                          jenkinsssh soft core unlimited
-                          jenkinsssh hard core unlimited
-                          EOF
-                          ssh-keyscan github.com github.com >> /home/jenkinsssh/.ssh/known_hosts
-                          ssh-keygen -F github.com -f /home/jenkinsssh/.ssh/known_hosts # verifies key is correctly installed
-                          chown jenkinsssh:jenkinsssh /home/jenkinsssh/.ssh/known_hosts
-                        installDocker: false
-                        installGit: false
-                        installMaven: false
-                        javaPath: java
-                        jvmOptions: "-Xms4G -Xmx4G -XX:+UseG1GC -XX:+UseCompressedOops -XX:+UseCompressedClassPointers\
-                      \ -XX:+AlwaysPreTouch -XX:+UseStringDeduplication -XX:+ParallelRefProcEnabled\
-                      \ -XX:+UnlockExperimentalVMOptions -XX:G1NewSizePercent=20 -XX:+UnlockDiagnosticVMOptions\
-                      \ -XX:G1SummarizeRSetStatsPeriod=1 -Djava.net.preferIPv4Stack=true"
-                        location: "UK South"
-                        noOfParallelJobs: 1
-                        osDiskSize: 60
-                        osDiskStorageAccountType: "Standard_LRS"
-                        osType: "Linux"
-                        retentionStrategy:
-                          azureVMCloudRetentionStrategy:
-                            idleTerminationMinutes: 60
-                        shutdownOnIdle: false
-                        storageAccountNameReferenceType: "existing"
-                        storageAccountType: "Standard_LRS"
-                        subnetName: "iaas"
+                      - templateName: "cnp-jenkins-builders"
                         templateDesc: "Jenkins build agents for HMCTS"
-                        templateDisabled: false
-                        templateName: "cnp-jenkins-builders"
-                        uamiID: /subscriptions/1497c3d7-ab6d-4bb7-8a10-b51d03189ee3/resourceGroups/managed-identities-cftsbox-intsvc-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/jenkins-cftsbox-intsvc-mi
                         usageMode: "NORMAL"
-                        usePrivateIP: true
-                        virtualMachineSize: "Standard_D4ds_v5"
-                        virtualNetworkName: "cft-ptlsbox-vnet"
-                        virtualNetworkResourceGroupName: "cft-ptlsbox-network-rg"
+                        <<: *vm_template_values_anchor
+                      - templateName: "cnp-jenkins-builders-daily"
+                        templateDesc: "Jenkins build agents for HMCTS daily builds"
+                        labels: "daily"
+                        noOfParallelJobs: 10
```

Comment:
we do not want parallel jobs, just run 1 per VM at a time

Reply:
removed, mixed this param up with maxVirtualMachinesLimit